tests/test_vault.py

```diff
@@ -0,0 +1,17 @@
+"""Made for use on dev machines in dev environment only."""
+import boto3
+import unittest
+
+from hris_publisher import vault
+
+
+class HRISTest(unittest.TestCase):
+    def setUp(self):
+        v = vault.Search(boto_session=None)
+        assert v is not None
+
+    def test_search_by_email(self):
+        boto_session = boto3.session.Session(region_name='us-west-2')
+        v = vault.Search(boto_session)
+        record = v.find_by_email('akrug@mozilla.com')
```

Comment:
while theres the comment about the test only running locally, this wont work with the stark email atm (since the test data uses `astark@mozilla.com` that is)

Reply:
These are designed to run in a scenario where someone has the Developer CIS role assumed against infosec-dev.  I'll update the doc accordingly.